changelog.json

```
@@ -1,5 +1,21 @@
 {
   "releases": {
+    "1.7.0-beta0": [
+      "[New] Repository list now displays recent repositories at top - #7132",
+      "[Fixed] Error when selecting diff text while diff is updating - #7131",
+      "[Fixed] Crash when unable to create log file on disk - #7096",
+      "[Fixed] Race condition with remote lookup could cause push to go to incorrect remote - #6986",
+      "[Fixed] Fix mistaken classification of all crashes as being related to launch - #7126",
+      "[Fixed] Improve user lookup to handle incomplete API response - #6945",
+      "[Fixed] Menus disabled when \"Merge Conflicts\" is launched for inactive repository - #7018",
```

I think?
```suggestion
      "[Fixed] \"Merge Conflicts\" flow should only be triggered for the active repository - #7018",
```

---

@billygriffin what do you think about:
```suggestion
      "[Fixed] Prevent menus from being disabled by activity in inactive repositories - #6313",
```